test/extensions/wasm/test_data/bad_signature_cpp.cc

```diff
@@ -3,9 +3,10 @@
 
 #include "proxy_wasm_intrinsics.h"
 
-extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onConfigure(uint32_t, int bad, char* configuration,
-                                                       int size) {
+extern "C" EMSCRIPTEN_KEEPALIVE uint32_t proxy_onConfigure(uint32_t, int bad, char* configuration,
```

Comment:
Nit: you could have left it as `void proxy_onConfigure(uint32_t, char* configuration, int size)`, since it's supposed to be bad signature anyway.

Reply:
Yes, but does it reverts the approval if I change anything... ?

Reply:
Probably.